components/BrandingContent/LogoSection/LogoSection.js

```diff
@@ -0,0 +1,226 @@
+import React from 'react';
+import Section from 'components/_common_/Section/Section';
+import styles from './logos.css';
+
+function LogoSection() {
+  return (
+    <Section title="Logo">
+      <p className={styles.logosInfo}>
+        The size ratio between the star and the medallion changes depending on the size of
+        reproduction. Please make use of the appropriate sized logo when creating collateral.
+      </p>
+      <p className={styles.logosInfo}>
+        In most cases, use the blue-accent version of the logo. The red-accent is delivered for
+        special uses only.
+      </p>
+      <p className={styles.logosInfo}>
+        <a href="https://s3.us-east-2.amazonaws.com/operationcode-web/Operation-Code-Logo.eps">
+          Download master EPS file
+        </a>
+      </p>
+      <p className={styles.logosInfo}>
+        The files below are transparent PNGs. Click-and-drag (or for mobile, press-and-hold) them
+        directly from your browser to download the file.
+      </p>
+      <div className={styles.logoImages}>
+        <section className={styles.sectionPadding}>
+          <h5>SMALL</h5>
+          <p>For use when the medal is between 0 and 1-inch tall.</p>
+          <div className={styles.marginSpace}>
+            <h6>LOGO</h6>
+            <div className={styles.smallLogos}>
+              <div className={styles.stackLogos}>
+                <img
+                  src="https://s3.amazonaws.com/operationcode-assets/branding/logos/small-blue-logo.png"
```

Comment:
Can you leverage `import { s3 } from 'common/constants/urls';` for all of these urls?

Example:
```suggestion
                  src={`${s3}branding/logos/small-blue-logo.png`}
```